docs/reference/elasticsearch/rest-apis/retrievers.md

```diff
@@ -233,10 +233,10 @@ Note, however, that wildcard field patterns will only resolve to fields that eit
 
 ### Limitations
 
-- **Single index**: Multi-field queries currently work with single index searches only
+- **Single index**: Applies only to the 9.1 version: multi-field queries work with single index searches only.
```

Comment:
these are the docs we added for the simplified retriever https://github.com/elastic/elasticsearch/pull/130559/

And this is the only mention I see of this limitation.
I wanted to see if I could use `applies_to` here but I couldn't find an example where we use it when we remove a limitation.
cc @leemthompo